gsa/src/web/components/form/checkbox.js

```diff
@@ -73,12 +101,6 @@ CheckboxComponent.propTypes = {
   onChange: PropTypes.func,
 };
 
-export default compose(
-  withLayout(),
-  withChangeHandler({
-    convert_func: convert_checked,
-    value_func: event => event.target.checked,
-  }),
-)(CheckboxComponent);
+export default compose(withLayout())(CheckboxComponent);
```

Comment:
You can drop the compose function if we only have one HOC left.

```suggestion
export default withLayout(CheckboxComponent);
```